subdomains/docs/_advanced/installers.md

```diff
@@ -74,3 +76,7 @@ Updating the PATH can be managed manually, if desired, or you can call `volta se
 ### Custom Volta Home (Optional)
 
 If you wish to use a different directory for the Volta data than the default `VOLTA_HOME` listed in the previous section, you need to set the environment variable `VOLTA_HOME` to that directory. If that is set, then `volta setup` will still work correctly for a custom data directory.
+
+## Github CI
+
+Volta distrubites a custom [Github action](https://github.com/volta-cli/action) that will automatically download Volta, cache it, and download the pinned versions of your tools in the project's `package.json`.
```

Comment:
This is a great addition! My only comment here is I think it's useful enough that we should move it earlier in the page. I would put this up above both `Custom Installers` and `Installing Old Versions`—right below the `Skipping Volta Setup` section.